2.4-rc/Dockerfile

```diff
@@ -50,7 +49,6 @@ RUN set -eux; \
 		USE_LUA=1 LUA_INC=/usr/include/lua5.3 \
 		USE_OPENSSL=1 \
 		USE_PCRE2=1 USE_PCRE2_JIT=1 \
-		USE_ZLIB=1 \
```

Comment:
Should you add `USE_SLZ=1` or is it enabled by default?

Reply:
Per https://github.com/haproxy/haproxy/commit/12840be, it appears to be the default if neither is specified. :smile:

Reply:
Ah, it's enabled by default per https://github.com/haproxy/haproxy/commit/12840be

I also opened a PR with the haproxytech image that I'm using here, https://github.com/haproxytech/haproxy-docker-debian/pull/4, which uses SLZ, and left USE_SLZ=1. I suppose I should remove that given the new default.